presto-iceberg/pom.xml

```diff
@@ -13,7 +13,7 @@
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
         <dep.iceberg.version>1.0.0</dep.iceberg.version>
-        <dep.nessie.version>0.43.0</dep.nessie.version>
+        <dep.nessie.version>0.44.0</dep.nessie.version>
```

Comment:
`NessieContainer` should probably also be updated

Reply:
It was not bumped for the previous Nessie release. 

So, I assumed, it has to be bumped with the Iceberg update. 

I have updated it now anyways.  I think it is still compatible. waiting for CI result.

Reply:
The Latest Nessie version is compatible with both Iceberg 1.0.0.
https://github.com/projectnessie/nessie#compatibility

So, it is ok to bump the test containers.